vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageVoiceBroadcastListeningItem.kt

```diff
@@ -122,10 +122,17 @@ abstract class MessageVoiceBroadcastListeningItem : AbsMessageVoiceBroadcastItem
 
     private fun bindSeekBar(holder: Holder) {
         with(holder) {
-            durationView.text = formatPlaybackTime(duration)
+            var remainingTimeText = formatPlaybackTime(duration)
+            remainingTimeView.text = if (duration < 1000) remainingTimeText else String.format("-%s", remainingTimeText)
```

Comment:
This seems to be duplicated below, just using different inputs. Would it make sense to create a helper function for it?